CONTRIBUTING.md

```diff
@@ -255,11 +261,10 @@ For most development work, the account will be `cucumber`, which is created when
 
 Conjur has `rspec` and `cucumber` tests.
 
-Note on perfomance testing: [ci/docker-compose.yml](ci/docker-compose.yml) and
-[conjur/ci/authn-k8s/dev/dev_conjur.template.yaml](conjur/ci/authn-k8s/dev/dev_conjur.template.yaml)
-set `WEB_CONCURRENCY: 0` a configuration that is useful for recording accurate
-coverage data, but isn't a realistic configuration, so shouldn't be used for
-benchmarking.
+Note on performance testing: set `WEB_CONCURRENCY: 0` - WEB_CONCURRENCY: 0 is a configuration that is useful for 
+recording accurate coverage data that can be used in the[ci/docker-compose.yml](ci/docker-compose.yml) and
```

Comment:
Some spelling errors:

in [ci/docker-compose.yml](ci/docker-compose.yml)

This isn't a realistic configuration and should not be used this for benchmarking.